### DISCUSSION TITLE
Guppy screen Tmc autotune

### DISCUSSION THREAD

**Discussion by EinsPommes** [Guilouz/Creality-Helper-Script-Wiki#509] [Issues]
Hi wanted to ask how tmc autotune works because it is greyed out

**Comment by Guilouz** [comment]
Make sure you have latest guppyscreen version.

**Comment by RyanSA** [comment]
I'm running the latest Guppyscreen version (0.0.26-beta) and the TMC autotune is still greyed out. What can I do to resolve?
